src/Mod/PartDesign/Gui/TaskPrimitiveParameters.cpp

```diff
@@ -811,103 +811,103 @@ void TaskBoxPrimitives::onWedgeZmaxChanged(double v)
+                                  ui->ellipsoidRadius3->value().getSafeUserString(),
+                                  ui->ellipsoidAngle1->value().getSafeUserString(),
+                                  ui->ellipsoidAngle2->value().getSafeUserString(),
+                                  ui->ellipsoidAngle3->value().getSafeUserString());
                 break;
 
             case 6:  // torus
-                cmd = QString::fromLatin1("%1.Radius1='%2'\n"
-                                          "%1.Radius2='%3'\n"
-                                          "%1.Angle1='%4'\n"
-                                          "%1.Angle2='%5'\n"
-                                          "%1.Angle3='%6'\n")
-                          .arg(name,
-                               ui->torusRadius1->value().getSafeUserString(),
-                               ui->torusRadius2->value().getSafeUserString(),
-                               ui->torusAngle1->value().getSafeUserString(),
-                               ui->torusAngle2->value().getSafeUserString(),
-                               ui->torusAngle3->value().getSafeUserString());
+                cmd = fmt::format("{0}.Radius1='{1}'\n"
+                                  "{0}.Radius2='{2}'\n"
+                                  "{0}.Angle1='{3}'\n"
+                                  "{0}.Angle2='{4}'\n"
+                                  "{0}.Angle3='{5}'\n",
+                                  name,
+                                  ui->torusRadius1->value().getSafeUserString(),
+                                  ui->torusRadius2->value().getSafeUserString(),
+                                  ui->torusAngle1->value().getSafeUserString(),
+                                  ui->torusAngle2->value().getSafeUserString(),
+                                  ui->torusAngle3->value().getSafeUserString());
                 break;
             case 7:  // prism
-                cmd = QString::fromLatin1("%1.Polygon=%2\n"
-                                          "%1.Circumradius='%3'\n"
-                                          "%1.Height='%4'\n"
-                                          "%1.FirstAngle='%5'\n"
-                                          "%1.SecondAngle='%6'\n")
-                          .arg(name,
-                               QString::number(ui->prismPolygon->value()),
-                               ui->prismCircumradius->value().getSafeUserString(),
-                               ui->prismHeight->value().getSafeUserString(),
-                               ui->prismXSkew->value().getSafeUserString(),
-                               ui->prismYSkew->value().getSafeUserString());
+                cmd = fmt::format("{0}.Polygon=%2\n"
+                                  "{0}.Circumradius='{2}'\n"
+                                  "{0}.Height='{3}'\n"
+                                  "{0}.FirstAngle='{4}'\n"
+                                  "{0}.SecondAngle='{5}'\n",

```

Comment:
The prism code is wrong and raises a syntax error. `Polygon={1}\n"`

Reply:
Good catch! Fixed. Does your IDE spot errors like this? I'd like to avoid such errors in the future.